### ISSUE TITLE
Metadata information cannot display it in the UI

### ISSUE THREAD

**Issue by sagarshrestha24** [kubeflow/metadata#256]
**What steps did you take and what happened:**
I have inserted most of the information but in artifact UI only few properties are shown.
some of the properties are not shown that is
                Accuracies & loss graphs 
                Data Version
                Training Logs
                Models
                Description
                Purpose
                Scientist ID 
                Architecture Used
                Keras & Tensorflow version
                Optimization algorithm
                Dataset Details (count, source, type, storage link)    
                Sample Images (real and fake)
                Base model
                Team members

**What did you expect to happen:**
I want to add all list item and it will show on the artifact store UI 

**Anything else you would like to add:**
![image (1)](https://user-images.githubusercontent.com/76894861/113252026-fffd8100-92e2-11eb-95cd-81777ca801ab.png)
here only specific properties are shown on artifact store UI and i want to add logs and more information 

- Metadata version:
- Kubeflow version:1.0.0
